cypress/e2e/group3/mytools.ts

```diff
@@ -275,11 +275,15 @@ describe('Dockstore my tools', () => {
       // Make sure page is loaded first
       cy.get('#tool-path').should('be.visible');
       cy.get('#register_tool_button').click();
+      // TODO: Fix this.  When 'Next' is clicked too fast, the next step is empty
+      cy.wait(1000);
```

Comment:
Copied from myworkflows.ts. Not ideal to have a wait, but I think this was the main fix that allowed the test to properly type in the source code repository input.